<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlterDialog</class>
 <widget class="QDialog" name="AlterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="1">
        <widget class="QTextEdit" name="textEdit_choice_optionC"/>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="combo_choice">
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>答案解析：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>正确答案：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>选项B：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QTextEdit" name="textEdit_choice_analysis"/>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="textEdit_choice_question"/>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="textEdit_choice_optionD"/>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="textEdit_choice_optionB"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>选项D：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>题目描述：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="textEdit_choice_optionA"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>选项A：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>选项C：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>分值：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="spinBox_choice_score"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>正确答案：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>答案解析：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QScrollArea" name="scrollArea">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>600</width>
            <height>207</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="4" column="0">
            <widget class="QLabel" name="label_blank5">
             <property name="text">
              <string>空白5：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_blank4"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_blank3">
             <property name="text">
              <string>空白3：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_blank2"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_blank1"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_blank2">
             <property name="text">
              <string>空白2：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_blank1">
             <property name="text">
              <string>空白1：</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_blank5"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_blank3"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_blank4">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>空白4：</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="textEdit_completion_analysis"/>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="textEdit_completion_question"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>题目描述：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>分值：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBox_completion_score"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="1">
        <widget class="QTextEdit" name="textEdit_program_describe"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>测试案例：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_program_num">
         <property name="text">
          <string>题目描述：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>617</width>
            <height>272</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QVBoxLayout" name="vLayout_caseContainer"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="tBtn_addCase">
               <property name="text">
                <string>添加案例</string>
               </property>
               <property name="icon">
                <iconset resource="res.qrc">
                 <normaloff>:/resource/add.png</normaloff>:/resource/add.png</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>分值：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_program_score"/>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btn_userFolder">
           <property name="text">
            <string>模板文件</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/resource/resize.png</normaloff>:/resource/resize.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_update">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
